antarest/core/exceptions.py

```diff
@@ -29,6 +29,11 @@ def __init__(self, message: str) -> None:
         super().__init__(EXPECTATION_FAILED, message)
 
 
+class CommandUpdateAuthorizationError(HTTPException):
+    def __init__(self, message: str) -> None:
+        super().__init__(EXPECTATION_FAILED, message)
```

Comment:
Use HttpStatus.LOCKED which is more semantically correct.
Also where EXPECTATION_FAILED is used, use the one from HttpStatus